pkgs/development/php-packages/psysh/default.nix

```diff
@@ -19,7 +19,7 @@ let
   composerLock = fetchurl {
     name = "composer.lock";
     url = "https://github.com/bobthecow/psysh/releases/download/v${version}/composer-v${version}.lock";
-    hash = "sha256-ur6mzla3uXeFL6aEHAPdpxGdvcgzOgTLW/CKPbNqeCg=";
+    hash = "sha256-PQDWShzvTY8yF+OUPVJAV0HMx0/KnA03TDhZUM7ppXw=";
```

Comment:
I still don't understand how the composer builder can mess up the hash of a `fetchurl`. I still think we should double-check what the hash change here has caused.